src/Generators/smallgenerators.jl

```diff
@@ -0,0 +1,25 @@
+abstract type SmallGraph <: GraphGenerator end
+
```

Comment:
SmallGraphs. The implementation is graph type dependent, so we have dispatches in `SimpleGraphs/Generators/smallgraphs.jl` for `SimpleGraph(gen::Bull)`, etc.